src/main/java/seedu/workbook/logic/parser/WorkBookParser.java

```diff
@@ -6,15 +6,7 @@
 import java.util.regex.Matcher;
 import java.util.regex.Pattern;
 
-import seedu.workbook.logic.commands.AddCommand;
-import seedu.workbook.logic.commands.ClearCommand;
-import seedu.workbook.logic.commands.Command;
-import seedu.workbook.logic.commands.DeleteCommand;
-import seedu.workbook.logic.commands.EditCommand;
-import seedu.workbook.logic.commands.ExitCommand;
-import seedu.workbook.logic.commands.FindCommand;
-import seedu.workbook.logic.commands.HelpCommand;
-import seedu.workbook.logic.commands.ListCommand;
+import seedu.workbook.logic.commands.*;
```

Comment:
Let's not use star imports as mentioned in the 2103T textbook

Reply:
Okay, will fix this.